tests/playbooks/tests_default.yml

```diff
@@ -4,3 +4,4 @@
   hosts: all
   roles:
     - linux-system-roles.network
+
```

Comment:
Please keep this file in the tests directory since it is needed for molecule.

Reply:
i have changed the [path to tests](https://github.com/a-mere-peasant/network/blob/master/molecule/default/molecule.yml#L26) should i change it back and fix everything accordingly or just let everything stay ?

Reply:
yes, please change it back.